src/hash/rescue/rpx/mod.rs

```diff
@@ -68,6 +71,9 @@ pub type CubicExtElement = CubeExtension<Felt>;
 /// the bottleneck for the security bound of the sponge in overwrite-mode only when it is
 /// lower than 2^128, we see that the target 128-bit security level is maintained as long as
 /// the size of the domain identifier space, including for padding, is less than 2^128.
+///
+/// ## Hashing of empty lists
+/// The current implementation hashes empty lists to the zero digest [0, 0, 0, 0].
```

Comment:
I would probably call this "Hashing empty input" or maybe "Hashing empty strings" and would also expand this a bit to mention the motivation - i.e., on empty input we do not apply a permutation.

Reply:
Changed